hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/paging/AlertComparator.java

```diff
@@ -98,6 +98,13 @@ public int compare(Alert o1, Alert o2) {
             return -1;
         }
         int iOrder = direction == Order.Direction.ASCENDING ? 1 : -1;
+        /*
+            Using tenant comparator first
+         */
+        int tenantComparator = o1.getTenantId().compareTo(o2.getTenantId());
+        if (tenantComparator != 0) {
+            return tenantComparator * iOrder;
+        }
```

Comment:
Yes.
First it orders by tenant and then it continues with the regular ordering by supporting fields.